### ISSUE TITLE
extend the set of types returned from from_type(type)

### ISSUE THREAD

**Issue by MarcoFavorito** [HypothesisWorks/hypothesis#1685]
Using version: `hypothesis==3.82.1`

Currently, the `from_type()` strategy with `type` as input returns:

```python
>>> from hypothesis.strategies import from_type
>>> from_type(type)
sampled_from([NoneType, bool, int, str, list, set, dict])
```

However, there are other built-in types that are missing, e.g. `tuple`, `bytes`.
A comprehensive list can be found [here](https://docs.python.org/3.7/library/stdtypes.html)

Is it a choice design to limit the types to the ones in the list above? 
It would be nice to see other types, like the mentioned `tuple`.

**Comment by Zac-HD** [comment]
We'd be very happy to accept a pull request to extend the list with as many built-in types as you would like to add :smile: 

The only real design constraint is that for consistency I'd keep it as a static list, and ordered from "simpler" to "more complicated" types.  The list is defined here:

https://github.com/HypothesisWorks/hypothesis/blob/da43eba90c1fbfa436c02e621f56dfa3a153a093/hypothesis-python/src/hypothesis/searchstrategy/types.py#L149

**Comment by eduzen** [comment]
Hi, It's my first time here. Could I pick this one?

**Comment by Zac-HD** [comment]
Closed by #1687
